package.json

```diff
@@ -25,25 +25,25 @@
   },
   "homepage": "https://github.com/pnarielwala/gulp-sloc2",
   "dependencies": {
-    "mkdirp": "^0.4.0",
-    "lodash": "^2.4.1",
-    "sloc": "^0.1.11",
-    "event-stream": "^3.1.2",
-    "gulp-util": "^2.2.14"
+    "gulp-util": "^3.0.8",
+    "lodash": "^4.17.11",
+    "mkdirp": "^0.5.1",
+    "sloc": "^0.2.1"
```

Comment:
The dependency order changed here, due to using npm to manage updates to the package.json - it sorts everything by default.